### DISCUSSION TITLE
Is it possible to improve the shadow?

### DISCUSSION THREAD

**Discussion by Hofknecht** [Hofknecht/SystemTrayMenu#345] [Q&A]
Is it possible to improve the shadow with that?
https://stackoverflow.com/questions/8793445/windows-7-style-dropshadow-in-borderless-form

Does anyone notice that the shadow isn't perfect?

Do we have some performance issues then?
Can we use uwp gpu features in winforms?

**Comment by chip33** [comment]
This would be a nice feature.
I did notice that STM looked flat but did not realise it was because of the dropshadow.

**Comment by Hofknecht** [comment]
Yes, that's also because we have a borderless form and draw a border, the rounded corners and the shadow ourselves. It would be cool to see how this would turn out in another framework, maybe we'll write everything again later in wpf
